apps-devstg/us-east-1/databases-mysql --/config.tf

```diff
@@ -2,10 +2,11 @@
 # AWS Provider Settings       #
 #=============================#
 provider "aws" {
-  region                   = var.region
-  profile                  = var.profile
-  shared_credentials_files = ["~/.aws/${var.project}/credentials"]
-  shared_config_files      = ["~/.aws/${var.project}/config"]
+  region  = var.region
+  profile = var.profile
+  # uncomment this 2 lines for aws sso disable
+  # shared_credentials_files = ["~/.aws/${var.project}/credentials"]
+  # shared_config_files      = ["~/.aws/${var.project}/config"]
```

Comment:
@rodriguez-matias Matu are this lines still present here but commented? In that case let's just remove them :) Thanks